examples/article.html

```diff
@@ -0,0 +1,93 @@
+---
+title: "Article Page"
+altLangPage: "article-fr.html"
+dateModified: "2025-08-12"
+description: "Office of the Auditor General of Canada Article page"
+language: "en"
+pageclass: "oag-theme wb-prettify all-pre"
+layout: no-container
+---
+
+<div class="container">
+	<div class="row my-4">
+		<div class="col-md-8">
+			<hgroup>
+				<p><span class="label label-default mr-3">Media Advisory</span><time datetime="2025-03-05">5 March 2025</time></p>
+				<h1 property="name" id="wb-cont">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat. Suspendisse est.</h1>
+			</hgroup>
+		</div>
+		<div class="col-md-4">
+			<div class="oag-brdr oag-rds p-3">
+				<section class="gc-followus">
```

Comment:
I think we should move this section to the sidebar and remove the grid, so just keep the `<hgroup>`.